README.md

```diff
@@ -1,30 +1,137 @@
 # spanner-sql-udf
 
-This repository contains a set of Spanner User-Defined Functions (UDFs) for a
-number of MySQL built-in functions. Combined together with Spanner's existing
-built-in functions, they provide coverage for most of MySQL's built-in
-functions. These UDFs are designed to assist customers migrating applications
-from MySQL to Spanner by providing definitions for MySQL built-in functions not
-already supported in Spanner, reducing the effort to migrate application code
-and queries.
+This repository contains a package of Spanner User-Defined Functions (UDFs) for
+many MySQL built-in functions. Combined with Spanner's existing built-in
+functions, these UDFs provide coverage for most of MySQL's built-in functions.
+When you migrate from MySQL to Spanner, you can use these UDFs to use MySQL
```

Comment:
Suggest:
When migrating from MySQL to Spanner, these UDFs allow you to continue using MySQL functions not natively supported by Spanner, significantly reducing the effort required to adapt your application code and queries.